Extensions/PlatformBehavior/Extension.cpp

```diff
@@ -475,6 +475,20 @@ void DeclarePlatformBehaviorExtension(gd::PlatformExtension& extension) {
         .SetFunctionName("SimulateLadderKey")
         .SetIncludeFile("PlatformBehavior/PlatformerObjectRuntimeBehavior.h");
 
+    aut.AddAction(
+           "SimulateReleaseLadderKey",
+           _("Simulate release ladder key press"),
+           _("Simulate a press of the Release Ladder key (used to get off a ladder)."),
+           _("Simulate pressing Release Ladder key for _PARAM0_"),
+           _("Controls"),
+           "res/conditions/keyboard24.png",
+           "res/conditions/keyboard.png")
+        .AddParameter("object", _("Object"))
+        .AddParameter("behavior", _("Behavior"), "PlatformerObjectBehavior")
+        .MarkAsAdvanced()
+        .SetFunctionName("SimulateReleaseLadderKey")
+        .SetIncludeFile("PlatformBehavior/PlatformerObjectRuntimeBehavior.h");
```

Comment:
Nitpicking: you can remove these two lines (used previously by GDCpp, and would be overwritten anyway in JsExtension.cpp)